numba/annotations/type_annotations.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import print_function, absolute_import
+import inspect
+import re
+from collections import Mapping, defaultdict, OrderedDict
```

Comment:
OrderedDict isn't provided in 2.6. We could add a backport in numba.utils though.